src/main/java/salesforce/ui/pages/legalentity/NewLegalEntityPage.java

```diff
@@ -0,0 +1,156 @@
+package salesforce.ui.pages.legalentity;
+
+import core.selenium.WebElementAction;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.entities.legalentity.LegalEntity;
+import salesforce.ui.pages.BasePage;
+import java.util.HashMap;
+import java.util.Set;
+import static salesforce.utils.Translator.translateValue;
+
+/**
+ * Interacts with the New Legal Entity elements.
+ */
+public class NewLegalEntityPage extends BasePage {
+
+    private WebElementAction webElementAction = new WebElementAction();
+
+    @FindBy(xpath = "//div/textarea[@class=\" textarea\"]")
```

Comment:
is not necessary to use the \" you can use '

Reply:
okay 👍🏻